src/main/java/core/basesyntax/service/operations/ReturnOperation.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public class ReturnOperation implements OperationHandler {
+
+    @Override
+    public void getOperation(List<FruitTransaction> transactions) {
+        List<FruitTransaction> transactionsSelected = transactions.stream()
+                .filter(a -> a.getOperation() == FruitTransaction.Operation.RETURN)
+                .toList();
+        for (FruitTransaction fruitTransaction : transactionsSelected) {
+            int newValue =
+                    Storage.fruits.get(fruitTransaction.getFruit())
+                            - fruitTransaction.getQuantity();
+            Storage.fruits.put(fruitTransaction.getFruit(), newValue);
+        }
+    }

```

Comment:
The method name `getOperation` is misleading as it does not return anything but performs an operation. Consider renaming it to reflect the action it performs, such as `apply` or `execute`.